### ISSUE TITLE
[A11y][WCAG2.1][Contrast]The color contrast ratio for the stars under "Stars" column is < 3:1

### ISSUE THREAD

**Issue by ryuyu** [NuGet/NuGetGallery#8286]
Copied from https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1069750

# Environment Details:
#URL:https://www.nuget.org/​
 ​
## OS Details:​
Windows 10 Version 1909 Build 18363.657​
​
## Browser and Version:​
Chromium Edge Version 80.0.361.51​
​
# Repro Steps:​
1.Hit the URL "https://www.nuget.org" 
2.Tab till "Packages" link in the top navigation and press enter​
3.Tab till any existing package and press enter​
4.Tab till "GitHub Usage" section and press enter​
5.Using color contrast analyzer, verify the color contrast ratio for the stars under "Stars" column header
​
## Actual Result:​
The color contrast ratio for the stars under "Stars" column is 2.51:1 which is < 3:1

## Expected Result:​
The color contrast ratio for non-text content should be >= 3:1
​
## User Impact:​
Users with visual impairments will find it difficult to access the content
​
# Notes:
Depending on how these stars are rendered, this will either be very simple, or very not simple.
